graphql/e2e/common/mutation.go

```diff
@@ -3003,7 +3003,7 @@ func addMultipleMutationWithOneError(t *testing.T) {
 	require.NoError(t, err)
 
 	require.Contains(t, gqlResponse.Errors[0].Error(),
-		`because ID "0x0" isn't a Author`)
+		`Invalid UID with value 0 for key`)
```

Comment:
Why did we need to change this error. This does not seem right. The error seems to be coming from x/keys.go: Parse function. The error should not change.

Reply:
Earlier as we were using `@filter(type(Author))`, it was querying the index key and then returning `because ID "0x0" isn't a Author` error because the UID=0 does not exist in the index (Value of the index key does not contain UID 0).
But now, we will be querying `dgraph.type`, it will construct a key that has UID=0 set, hence that key will fail at the time of parsing the key.

Reply:
As discussed over call, lets keep the error message as before while changing the invalid author ID.